src/Targets/Microsoft.CSharp.DesignTime.targets

```diff
@@ -17,8 +18,18 @@
 
   <!-- Project Capabilities -->
   <ItemGroup>
-    <ProjectCapability Include="AppDesigner"/>
-    <ProjectCapability Include="ManagedLang"/> <!-- Temporary: See https://github.com/dotnet/roslyn-project-system/issues/47 -->
+    <ProjectCapability Include="AppDesigner" />
+    <ProjectCapability Include="ManagedLang" /> <!-- Temporary: See https://github.com/dotnet/roslyn-project-system/issues/47 -->
+    <ProjectCapability Include="LanguageService;
+                                ProjectConfigurationsInferredFromUsage;
+                                AssemblyReferences;
+                                COMReferences;
+                                ProjectReferences;
+                                SharedProjectReferences;
+                                OutputGroups;
+                                AllTargetOutputGroups;
+                                VisualStudioWellKnownOutputGroups;
+                                SingleFileGenerators;" />
```

Comment:
All changes in this file i will apply also to VB target, so far i tested only for cs.